inlang/source-code/paraglide/paraglide-js-adapter-unplugin/src/index.ts

```diff
@@ -0,0 +1,48 @@
+import { createUnplugin } from "unplugin"
+import { loadProject } from "@inlang/sdk"
+import { exec } from "node:child_process"
+import path from "node:path"
+import fs from "node:fs/promises"
+
+type UserConfig = {
+	project: string
+	outdir: string
+	timeout?: number
+	onInit?: boolean
+}
+
+export const paraglide = createUnplugin((config: UserConfig) => {
+	const options = {
+		silent: false,
+		onInit: true,
+		...config,
+	}
+
+	const execute = () => {
+		exec(
+			`paraglide-js compile --project ${options.project} --outdir ${options.outdir}`,
```

Comment:
hmm interesting. i would have called the `compile` directly and avoided the CLI. 

Are you certain that calling the binary will work in every environment? If not, calling `compile` directly seems like a safer option

Reply:
I implemented it like that without further thinking about it. Calling compile directly makes probably more sense.

Reply:
@LorisSigrist I recommend directly using the compile function from paraglide. What do you think?